User.py

```diff
@@ -95,3 +95,14 @@ def addExercise(self, exercise, reps):
     def hasDoneExercise(self, exercise):
         return exercise["id"] in self.exercise_counts
 
+
+def parseSlackJSON(response):
+    isMessageOkay = False
+    try:
+        if response.ok:
+            parsed_message = json.loads(response.text, encoding='utf-8')
+    except:
+        print "Caught exception parsing response status: " + str(response.status_code) + ", text: " + response.text
+
+    isMessageOkay = response.ok & parsed_message["ok"]
```

Comment:
I don't think Python has an && operator. Doing & is a bitwise operator. So, evaluating that on true/false should be fine.